scripts.js

```diff
@@ -26,7 +28,7 @@ var dataDocumentsSheet = '1VZmrdC-rm6noqxMoFWiPimOiM3-zmhk5kOmJ8RppU9w';
 var dataDocuments = new Map([
     ['2019', '18JTbEUmTiUCMbazqj593Q6QNfpQZbjSiKUpHrWF9EGc'],
     ['2020', '1aPQuyvb8Y1SH37kkD1eVFftkscHB63cnU92HQeuR9n4'],
-    ['2021', '1Puj3Apgo7cK5AD-x25RTvHUbs9yGzdtBZUFsp13-z0k']
+    ['2021', '1Puj3Apgo7cK5AD-x25RTvHUbs9yGzdtBZUFsp13-z0k'],
```

Comment:
Commas on the last element scare me, but I respect it, especially here when we'll add another element every year.

Reply:
I think it's somewhat standard to use trailing commas in lists that span multiple lines. And yeah, it'll help when we inevitably need to add to the list.